packages/components/src/internal/components/ontology/OntologyBrowserPanel.tsx

```diff
@@ -0,0 +1,83 @@
+import React, { FC, useEffect, useState } from 'react';
+import { Col, Row } from 'react-bootstrap';
+
+import { LoadingSpinner } from '../base/LoadingSpinner';
+
+import { OntologyTabs } from './OntologyTabs';
+import { ConceptInformationTabs, ConceptInfoTabs } from './ConceptInformationTabs';
+import { getOntologyDetails } from './actions';
+import { ConceptModel, OntologyModel } from './models';
+
+export interface OntologyBrowserProps {
+    ontologyId?: number;
+}
+
+interface ConceptPathModel {
+    path: string;
+    conceptCode: string;
+    children: ConceptPathModel[];
+}
```

Comment:
Looks like this isn't used. Also, if it is needed, I'd move it to the models.ts file.